### PR TITLE
Test config and .repo file filenames containing invalid UTF-8 chars

### PR DESCRIPTION
**Pull request by pkratoch**

### CODE REVIEW

**Comment by pkratoch** [comment]
Requires libdnf that was built with swig >= 4.0.0 or swig containing certain patch (works on Fedora 31, doesn't work on Fedora 30, but that may change)
The last commit also requires https://github.com/rpm-software-management/dnf/pull/1599

File: dnf-behave-tests/common/lib/cmd.py [hunk 1]
```diff
@@ -33,27 +33,32 @@ def run_in_context(context, cmd, can_fail=False, **run_args):
...
+        cmd = context.lc_all + cmd
+
     context.cmd = cmd
 
     if hasattr(context.scenario, "working_dir") and 'cwd' not in run_args:
         run_args['cwd'] = context.scenario.working_dir
 
     context.cmd_exitcode, context.cmd_stdout, context.cmd_stderr = run(cmd, **run_args)
-
+
```
Comment (j-mracek) [inline dnf-behave-tests/common/lib/cmd.py]: @pkratoch Please remove empty spaces

Comment (pkratoch) [reply dnf-behave-tests/common/lib/cmd.py]: Fixed.

**Comment by j-mracek** [comment]
I am sorry, I wasn't an assignee but I merge it.

**Pull request merged**